wmlab/config.yaml

```diff
@@ -0,0 +1,102 @@
+defaults:
+    - override hydra/launcher: submitit_local
+
+# environment
+task: dog-run
+obs: state
+
+env:
+  _target_: dflex.envs.AntEnv
+  render: False
+  device: "cuda"
+  num_envs: 1
+  stochastic_init: True
+  no_grad: True
+  episode_length: 500
+  MM_caching_frequency: 16
+  early_termination: True
+
+# evaluation
+checkpoint: ???
+eval_episodes: 10
+eval_freq: 5000
+
+# training
+steps: 10_000_000
+seed_train_steps: 5000
+batch_size: 1024 # default 256. Mine: num_envs * 32
+reward_coef: 0.1
+value_coef: 0.1
+consistency_coef: 20
+rho: 0.5 # weight of predicted next states for policy loss
+lr: 3e-4
+enc_lr_scale: 0.3
+grad_clip_norm: 20
+actor_grad_clip_norm: 20
+tau: 0.01
+discount_denom: 5
+discount_min: 0.95
+discount_max: 0.995
+buffer_size: 1_000_000 # minimum must be 1000 x num_envs
+exp_name: default
+data_dir: /storage/home/hcoda1/7/igeorgiev3/git/FoWM/data/dflex-ant
```

Comment:
filepath on open-source repo is okay?

Reply:
Generally we should leave a placeholder for this in the final public version of this. However, for the time being and for moving fast, this is fine as long as we're on the same machine.